gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-bin.zip
```

Comment:
This doesn't seem to be complete as changes to the Gradle wrapper script are missing. Note that you need to run `./gradlew wrapper --gradle-version=8.3` at least twice to also update the wrapper itself.

Reply:
For reference, see https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper:

> Note that running the wrapper task once will update gradle-wrapper.properties only, but leave the wrapper itself in gradle-wrapper.jar untouched. This is usually fine as new versions of Gradle can be run even with ancient wrapper files. If you nevertheless want all the wrapper files to be completely up-to-date, you’ll need to run the wrapper task a second time.